app/src/pages/choose-clinic.tsx

```diff
@@ -1,112 +1,160 @@
-import { useAppContext } from '@context/state'
```

Comment:
Notes for reviewers: The ChooseClinic page (`/choose-clinic`) is an editable form page in the form wizard flow. The back link routes to either `/eligibility` or `/income` depending on data the user entered on `/eligibility` (this is just for reviewer context; the logic for this is not part of this commit). The call to action button routes the user to `/contact`. In order for a user to be allowed onto this page, they must have filled out all previous pages with valid data. 

![Screen Shot 2022-10-28 at 2 30 04 AM](https://user-images.githubusercontent.com/67701/198554770-537296a3-068f-4279-b3c1-e5eeff71276b.png)